app.py

```diff
@@ -4,81 +4,173 @@
 
 from aws_stack.vpc_stack import VpcStack
 from aws_stack.asg_stack import AsgStack
-import sys
-import os
 import yaml
-import argparse
+from botocore import client
+import boto3
+import sys
+from datetime import datetime
+from datetime import timedelta
+import logging
+
+feature_query_spot_prices_disabled = True
 
 
 def configure(stack_name: str):
-  defaults = {   
-                stack_name: {
-                  "region": "us-east-1",
-                  "ec2_instance_type": "c5n.large",
-                  "asg_size": 2,
-                  "max_spot_price": "",
-                }
-             }
-
-  try:
-      with open("config.yaml") as f:
-          user_config = yaml.safe_load(f)
-  except yaml.YAMLError as exc:
-      print("Failed parsing the configuration file, exiting", exc)
-      sys.exit(1)
-  except OSError as exc:
-      print("Unable to read configuration file, exiting", exc)
-      sys.exit(1)
+    logging.debug("Processing the configuration file")
+    defaults = {
```

Comment:
Eh. Is my `.editorconfig` broken? This shouldn't have changed spacing here.

Reply:
nah, my rebase screwed up the indents and this is correct